### ISSUE TITLE
 access to path '/home/teto/nixpkgs' is forbidden in restricted mode

### ISSUE THREAD

**Issue by teto** [NixOS/nix#3079]
I've recently switched to use nixFlake but I can't run nix repl with my own nixpkgs anymore.
I haven't been able to find how to get back the previous behavior.

nix (Nix) 2.3pre20190712_aa82f8b

```
cd nixpkgs
nix repl --option restrict-eval false . ./nixos
error: access to path '/home/teto' is forbidden in restricted mode
```
Maybe the error could include the option name to work around the issue.

**Comment by edolstra** [comment]
Fixed in 2dbd69dbf4538d5b7947d192979ff4feab322c2e.

**Comment by teto** [comment]
ty
